tests/Unit/ConfigurationTest.php

```diff
@@ -294,6 +334,23 @@ public static function dataTestConfigurationTree(): iterable
         ];
     }
 
+    public static function dataTestSettingsDynamicCheckerInvalid(): iterable
```

Comment:
```suggestion
    /**
     * @return iterable<array{value: mixed}>
     */
    public static function dataTestSettingsDynamicCheckerInvalid(): iterable
```

Reply:
Good catch! I will fix it as well as the PHP-CS-Fixer issues later.

Reply:
Fixed